app/istqb-ct-ai-complete-guide/istqb-ct-ai-complete-guide.css

```
@@ -0,0 +1,999 @@
+                from {
+                    opacity: 0;
+                    transform: translateY(18px);
+                }
+                to {
+                    opacity: 1;
+                    transform: translateY(0);
+                }
+            }
+            @keyframes grow-bar {
+                from {
+                    width: 0 !important;
```

Stylelint (17.9.0)

[error] Disallowed !important (keyframe-declaration-no-important)

(keyframe-declaration-no-important)